docs/spec/relayer/Relayer.tla

```diff
@@ -155,14 +77,17 @@ LightClientUpdates(srcChainID, dstChainID, relayer) ==
 \* Compute connection datagrams designated for dstChainID. 
 \* These are used to update the connection end on dstChainID in the environment.
 ConnectionDatagrams(srcChainID, dstChainID) ==
-    LET srcConnectionEnd == GetConnectionEnd(srcChainID) IN
-    LET dstConnectionEnd == GetConnectionEnd(dstChainID) IN
+    LET srcChain == GetChainByID(srcChainID) IN
+    LET dstChain == GetChainByID(dstChainID) IN
+    
+    LET srcConnectionEnd == GetConnectionEnd(srcChain) IN
+    LET dstConnectionEnd == GetConnectionEnd(dstChain) IN
 
     LET srcConnectionID == GetConnectionID(srcChainID) IN
     LET dstConnectionID == GetConnectionID(dstChainID) IN
 
-    LET srcHeight == GetLatestHeight(srcChainID) IN
-    LET srcConsensusHeight == GetMaxCounterpartyClientHeight(srcChainID) IN
+    LET srcHeight == GetLatestHeight(srcChain) IN
+    LET srcConsensusHeight == GetMaxCounterpartyClientHeight(srcChain) IN
```

Comment:
Shouldn't it be dstChain?Also maybe better using the same name as above (srcClientHeight).

Reply:
Yes I tried before, in spec and code to find a good name for this:
"the height of the latest consensus of clientB running on chain A" or
"the height of the latest consensus of dest client running on src chain".
First thing that comes to my mind here is that this is a height of chain B that you obtain from its client on A. So indeed `srcConsensusHeight` might make someone think that this is a height of chain A.